libs/galley-types/test/unit/Test/Galley/Types.hs

```diff
@@ -28,6 +28,7 @@ import Imports
 import Test.Tasty
 import Test.Tasty.HUnit
 
+-- wire-api: we still need this
```

Comment:
I don't understand this comment.  Do you mean this?:

```suggestion
-- TODO: move this to wire-api
```

Reply:
ah, forgot to remove this one. It was just a reminder for myself that this test needs to stay where it is.